### PR TITLE
E2e/validation report

### LINKED ISSUE #1380: 021 - Validation report e2e test
- [ ] Update insert record test to add a new plot to cluster 5. (do not enter any values)
- [ ] go to validation report. expect 3 rows there in order they must be: Cluster[5] → Plot[] :Required value, Cluster[1] / Cluster decimal : cluster_decimal < 10, Cluster[2] / Cluster decimal : cluster_decimal < 20
- [ ] Click first row and enter 2 in plot id field.
- [ ] Go back to validation report and rows now become 4 
Cluster[5] / Plot[2] / Plot id :Duplicate entity key, 
Cluster[5] / Plot[2] / Plot id :Duplicate entity key, Cluster[1] / Cluster decimal : cluster_decimal < 10, Cluster[2] / Cluster decimal : cluster_decimal < 20
- [ ] Click second row and enter 3 into plot id value
- [ ] Go back to validation report and rows now become 2 
Cluster[1] / Cluster decimal : cluster_decimal < 10, Cluster[2] / Cluster decimal : cluster_decimal < 20
- [ ] Click second row and enter 11 in cluster_decimal field
- [ ] Go back to validation report and row nows become 1 
Cluster[1] / Cluster decimal : cluster_decimal < 10

### PR DESCRIPTION
**Pull request by ramirobg94**
resolves #1380

### CODE REVIEW

File: server/modules/record/repository/validationReportRepository.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ const query = (surveyId) =>
       h.record_uuid = r.uuid 
       AND r.cycle = $/cycle/
       AND NOT r.preview
-    ORDER BY r.uuid, h.node_id`
+    ORDER BY h.node_id, r.uuid`
```
Comment (minotogna) [inline server/modules/record/repository/validationReportRepository.js:48]: why this change ?

Comment (ramirobg94) [reply server/modules/record/repository/validationReportRepository.js:48]: because we ordered the errors by uuid which is random and the node_id is number created in order

I can leave the query as it was and then I have to change the test to be "order" agnostic

what do you think? @minotogna

Comment (minotogna) [reply server/modules/record/repository/validationReportRepository.js:48]: @ramirobg94 let's use r.date_added, h.node_id

**Review by minotogna — APPROVED** [review]
